Makefile

```diff
@@ -81,56 +73,6 @@ doc-man:
 doc-html:
 	(cd doc; make html)
 
-centos:
-	make clean
-	mkdir RHBUILD
-	mkdir -p RHBUILD/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
-	# create tarball
-	mkdir -p RHBUILD/SOURCES/privacyidea-${VERSION}
-	rsync -a --exclude=".*" --exclude="privacyIDEA.egg-info" --exclude="RHBUILD" --exclude="debian" --exclude="dist" --exclude="build" \
-                --exclude="tests" . RHBUILD/SOURCES/privacyidea-${VERSION} || true
-	# We are using the same config file as in debia an replace it in setup.py
-	cp deploy/centos/pi.cfg RHBUILD/SOURCES/privacyidea-${VERSION}/deploy/centos/
-	# pack the modified source
-	(cd RHBUILD/SOURCES/; tar -zcf privacyidea-${VERSION}.tar.gz privacyidea-${VERSION})
-	rm -fr RHBUILD/SOURCES/privacyidea-${VERSION}
-	# copy spec file
-	cp deploy/centos/privacyidea.spec RHBUILD/SPECS
-	# build it
-	rpmbuild --define "_topdir $(CURDIR)/RHBUILD" -ba RHBUILD/SPECS/privacyidea.spec
-
-
-debianize:
-	make clean
-	make doc-man
-	mkdir -p DEBUILD/privacyidea.org/debian
-	cp -r ${SRCDIRS} ${SRCFILES} DEBUILD/privacyidea.org || true
-	# remove the requirement for pyOpenSSL otherwise we get a breaking dependency for trusty
-	grep -v pyOpenSSL setup.py > DEBUILD/privacyidea.org/setup.py
-	# We need to touch this, so that our config files 
-	# are written to /etc
-	touch DEBUILD/privacyidea.org/PRIVACYIDEA_PACKAGE
-	cp LICENSE DEBUILD/privacyidea.org/debian/copyright
-	cp LICENSE DEBUILD/privacyidea.org/debian/python-privacyidea.copyright
-	cp LICENSE DEBUILD/privacyidea.org/debian/privacyidea-all.copyright
-	(cd DEBUILD; tar -zcf python-privacyidea_${SHORT_VERSION}.orig.tar.gz --exclude=privacyidea.org/debian privacyidea.org)
-	(cd DEBUILD; tar -zcf python-privacyidea_${VERSION}.orig.tar.gz --exclude=privacyidea.org/debian privacyidea.org)
-	(cd DEBUILD; tar -zcf python-privacyidea_${VERSION_JESSIE}.orig.tar.gz --exclude=privacyidea.org/debian privacyidea.org)
-	(cd DEBUILD; tar -zcf privacyidea-venv_${VERSION}.orig.tar.gz --exclude=privacyidea.org/debian privacyidea.org)
-
-builddeb-nosign:
-	make debianize
-	cp -r deploy/debian-ubuntu/* DEBUILD/privacyidea.org/debian/
-	sed -e s/"trusty) trusty; urgency"/"$(LOCAL_SERIES)) $(LOCAL_SERIES); urgency"/g deploy/debian-ubuntu/changelog > DEBUILD/privacyidea.org/debian/changelog
-	(cd DEBUILD/privacyidea.org; debuild -b -i -us -uc)
-
-builddeb:
-	make debianize
-	################## Renew the changelog
-	cp -r deploy/debian-ubuntu/* DEBUILD/privacyidea.org/debian/
-	sed -e s/"trusty) trusty; urgency"/"$(LOCAL_SERIES)) $(LOCAL_SERIES); urgency"/g deploy/debian-ubuntu/changelog > DEBUILD/privacyidea.org/debian/changelog
-	################# Build
-	(cd DEBUILD/privacyidea.org; debuild --no-lintian)
 
 venvdeb:
```

Comment:
shouldn't this be removed as well? The `debianize` target is already removed.

Reply:
The venvdeb is currently used for building debian packages as basis for UCS. 
But I think you are right, we should remove this, too, so that we have actually remove *all* distro dependent builds from the source tree. I think this makes it more clear.
Will do